### PR TITLE
[#135504943] Create main UI page using CardLayout

### PR DESCRIPTION
**Pull request by chunweng428**
Created a main page using CardLayout for card switching purpose and created hard coded sample data

### CODE REVIEW

**Comment by TeckYong** [comment]
what he meaning of USJ ? might use better naming ?

File: process/code/UJS.java [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    public static ArrayList<Customer> loadCustomer() {
+        ArrayList<Customer> c = new ArrayList<>();
+        c.add(new Customer("", "Chan Chun Weng", "999999-99-9999", "012-3456789", "TBR"));
+        c.add(new Customer("", "Teh Teck Yong", "999999-99-9999", "012-3456789", "TBR"));
+        c.add(new Customer("", "Yong Cheng Kiang", "999999-99-9999", "012-3456789", "TBR"));
+        c.add(new Customer("", "Kezin Siew", "999999-99-9999", "012-3456789", "TBR"));
+        c.add(new Customer("", "Lai Tau Leong", "999999-99-9999", "012-3456789", "TBR"));
+        c.add(new Customer("", "Yap Ming Han", "999999-99-9999", "012-3456789", "TBR"));
+        return c;
+    }
```
Comment (TeckYong) [inline process/code/UJS.java:166]: Are these are fake customer data for testing ?

Comment (chunweng428) [reply process/code/UJS.java:166]: Yes, it's for testing purpose only.

File: process/code/UJS.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+public class UJS extends javax.swing.JFrame {
+
+    protected static double goldPricePerGram = 175 + (200 - 175) * new Random().nextDouble();
+    protected static double silverPricePerGram = 2.63 + (5 - 2.63) * new Random().nextDouble();
+    protected static double diamondPricePerGram = 1800 + (2100 - 1800) * new Random().nextDouble();
+    protected static double jadePricePerGram = 50 + (60 - 50) * new Random().nextDouble();
+    protected static double platinumPricePerGram = 240 + (250 - 240) * new Random().nextDouble();
```
Comment (MasterODST) [inline process/code/UJS.java:14]: Question, are these the set prices of the material types based of current market costs??

Comment (chunweng428) [reply process/code/UJS.java:14]: Yes, it is approximate cost to each product and it only for testing purpose. I used random to generate the cost of each product.

**Comment by TeckYong** [comment]
What the meaning of UJS?

**Comment by TeckYong** [comment]
What the meaning of UJS ?

**Review by TeckYong — COMMENTED** [review]
What the meaning of UJS?

**Comment by TeckYong** [comment]
testing

**Review by TeckYong — APPROVED** [review]

**Review by tanjunlem — APPROVED** [review]
I think this coding should be worked as we planned. Well done.

**Review by MasterODST — APPROVED** [review]

**Pull request merged**